builder/test/image_spec.rb

```diff
@@ -47,11 +47,11 @@
     let(:stdout) { run_mounted("ls /var/local").stdout }
 
     it "Consul export file exists" do
-      expect(stdout).to contain('rpi-consul_v0.6.4.tar.gz')
+      expect(stdout).to contain('rpi-consul_0.6.4.tar.gz')
     end
 
     it "Docker Swarm export file exists" do
-      expect(stdout).to contain('rpi-swarm_v1.2.2.tar.gz')
+      expect(stdout).to contain('rpi-swarm_1.2.2.tar.gz')
```

Comment:
@firecyberice If we rename those files here we need to adjust the configuration in https://github.com/hypriot/image-builder-rpi/blob/master/builder/files/boot/device-init.yaml#L5-L6. Is there a reason why the renaming was necessary?